pil-analyzer/src/condenser.rs

```diff
@@ -1033,6 +1035,41 @@ fn try_value_to_expression<T: FieldElement>(value: &Value<'_, T>) -> Result<Expr
                     type_args: None,
                 }),
             ),
+
+            AlgebraicExpression::BinaryOperation(AlgebraicBinaryOperation { left, op, right }) => {
+                Expression::BinaryOperation(
+                    SourceRef::unknown(),
+                    BinaryOperation {
+                        left: Box::new(try_value_to_expression(&Value::<T>::Expression(
+                            (**left).clone(),
+                        ))?),
+                        op: match op {
```

Comment:
You should be able to just use `op.into()` here.

Reply:
How can I use into() here?

Reply:
Is it just `(*op).into()`?